ci-operator/step-registry/workers-rhel/ibmcloud-provision/workers-rhel-ibmcloud-provision-commands.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -xeuo pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
```

Comment:
I did not see much meaning of wrapper one command into the function.

Reply:
I merged ibmcloud_login and  checkCli to gether, if OK, I will update other *ibmcloud*-command.sh

> function ibmcloud_login {
  export IBMCLOUD_CLI=ibmcloud
  export IBMCLOUD_HOME=/output
  region="${LEASED_RESOURCE}"
  export region
  echo "Try to login..."
  "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
}

Reply:
I suggest not to place the lines in a function which is only called one time in the step, but directly run the lines. And in prow ci, we already set -o errexit, a function may skip the error happen in the function.

Reply:
in this function, export IBMCLOUD_CLI, IBMCLOUD_HOME and region. 
collect the login to a function in order to reuse it in other  \*ibmcloud\*-command.sh

Reply:
What means "reuse"? copy/paste from one commands shell script to another?